src/matter/cluster/WaterContentMeasurementCluster.ts

```diff
@@ -0,0 +1,62 @@
+/**
+ * @license
+ * Copyright 2022 The node-matter Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+
+import {Attribute, OptionalAttribute, Cluster, Command, TlvNoArguments, TlvNoResponse} from "./Cluster";
+import {BitFlag, MatterApplicationClusterSpecificationV1_0, TlvBitmap, TlvInt16, TlvUInt16, TlvEnum, TlvField, TlvNullable, TlvObject, TlvSchema, TlvUInt8} from "@project-chip/matter.js";
+
+/** @see {@link MatterApplicationClusterSpecificationV1_0} § 2.6.4 */
+const  attributes = {
+    /** Represents the water content in % as follows: MeasuredValue = 100 x water content */
+    measuredValue: Attribute(0, TlvNullable(TlvUInt16.bound({min: 0, max: 10000}))),
+
+    /** Indicates the minimum value of MeasuredValue that can be measured. */
+    minMeasuredValue: Attribute(1, TlvNullable(TlvUInt16.bound({min: 0}))),
+
+    /** Indicates the maximum value of MeasuredValue that can be measured. */
+    maxMeasuredValue: Attribute(2, TlvNullable(TlvUInt16.bound({max: 10000}))),
+
+    /** The magnitude of the possible error that is associated with MeasuredValue 
+    * attribute using the same unit */
```

Comment:
But then lets do proper :-)

```suggestion
    /** 
     * The magnitude of the possible error that is associated with MeasuredValue 
     * attribute using the same unit 
     */
```